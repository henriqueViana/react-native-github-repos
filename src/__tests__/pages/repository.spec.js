import React from 'react'
import { shallow } from 'enzyme'

import Repository from '../../pages/repository'
import MockNavigator from '../../__mocks__/mockNavigator'

let wrapper

describe('-----Repository-----', () => {

  beforeEach(() => {
    wrapper = shallow(<MockNavigator component={Repository} />)
  })

  it('should render correcty Rrepository page', () => {
    expect(wrapper).toMatchSnapshot()
  })
})