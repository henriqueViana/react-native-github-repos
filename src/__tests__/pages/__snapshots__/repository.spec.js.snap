// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`-----Repository----- should render correcty Rrepository page 1`] = `
<ForwardRef(NavigationContainer)>
  <StackNavigator>
    <Screen
      component={[Function]}
      initialParams={Object {}}
      name="MockedScreen"
    />
  </StackNavigator>
</ForwardRef(NavigationContainer)>
`;
